src/librustc_mir/borrow_check/nll/region_infer/mod.rs

```diff
@@ -125,6 +123,10 @@ pub(crate) enum Cause {
     },
 }
 
+pub(crate) struct RegionCausalInfo {
+    pub(crate) inferred_values: RegionValues,
```

Comment:
This field should not be public, I think -- `RegionValues` were meant to be encapsulated within the `region_infer` module. If we move the `why_borrow_contains_point` method onto the `RegionCausalInfo` struct, that would work out fine.